src/mission.cpp

```diff
@@ -2496,6 +2496,32 @@ Before we get into a major fight just make sure we have the gear we need, boss."
         }
         break;
 
+    case MISSION_FIND_ZSG:
+        if( state == "TALK_MISSION_DESCRIBE" ) {
+            return _("I've can an idea about how to improve our chances of surviving");
```

Comment:
Thanks for the feedback.

I've made the changes in my local tree to fix those two typos - how do I go about publishing them? Do I amend the commit and push the fix back to github or something else? I checked 
http://smf.cataclysmdda.com/index.php?topic=1063.0 but it didn't address the issue of making changes to your code after submitting the pull request.